userguide/tasks.rst

```diff
@@ -1793,23 +1791,37 @@ shows the configuration options for Cloud Syncs.
-   | Encryption password | string              | Only appears when :guilabel:`Remote encryption` is enabled. Enter the password to encrypt and decrypt   |
-   |                     |                     | remote data. *Warning:* Always save and back up this password. Losing the encryption password can       |
-   |                     |                     | result in data loss.                                                                                    |
+   |                     |                     | *PULL:* Decrypt files that are being stored on the remote system before the transfer. Transferring the  |
+   |                     |                     | encrypted files requires entering the same :guilabel:`Encryption Password` and                          |
+   |                     |                     | :guilabel:`Encryption salt` that was used to encrypt the files.                                         |
    |                     |                     |                                                                                                         |
+   |                     |                     | Adds the :guilabel:`Filename encryption`, :guilabel:`Encryption password`, and                          |
+   |                     |                     | :guilabel:`Encryption salt` options. Additional details about the encryption algorithm and key          |
+   |                     |                     | derivation are available in the                                                                         |
+   |                     |                     | `rclone crypt File formats documentation <https://rclone.org/crypt/#file-formats>`__.                   |
    +---------------------+---------------------+---------------------------------------------------------------------------------------------------------+
-   | Encryption salt     | string              | Only appears when :guilabel:`Remote encryption` is enabled. Enter a long string of random characters    |
-   |                     |                     | for use as `salt <https://searchsecurity.techtarget.com/definition/salt>`__ for the encryption          |
-   |                     |                     | password. *Warning:* Save and back up the encryption salt value. Losing the salt value can result in    |
-   |                     |                     | data loss.                                                                                              |
+   | Filename encryption | checkbox            | Encrypt (*PUSH*) or decrypt (*PULL*) file names with the rclone `"Standard" file name encryption mode   |
+   |                     |                     | <https://rclone.org/crypt/#file-name-encryption-modes>`__. The original directory structure is          |
+   |                     |                     | preserved. Identical file names remain identical after encryption.                                      |
```

Comment:
"File names remain intact after encryption" would make more sense. Identical file names remain Idential is redundant.